.github/workflows/main_ci.yml

```diff
@@ -4,9 +4,11 @@ on:
   push:
     branches:
       - pydarshan-devel
+      - "**"
```

Comment:
This isn't really needed since the CI will run on the PR itself from the change below.